configuration.rst

```diff
@@ -134,6 +135,17 @@ db-extra-search-path
 
   Multiple schemas can be added in a comma-separated string, e.g. ``public, extensions``.
 
+.. _db-prepared-statements:
+
+db-prepared-statements
+----------------------
+
+  Enables or disables prepared statements.
+
+  When disabled, the queries will be parameterized but they will not be prepared. In other words, the generated queries will be safe and invulnerable to SQL injection, but they will not be cached at the database server side. Not using prepared statements will noticeably decrease performance (around 23% decrease according to our benchmarks), so it's recommended to always have this setting enabled.
```

Comment:
The 23% decrease depends on the type of queries being sent, so I'm skipping that for when we have a more detailed performance page.

```suggestion
  When disabled, the generated queries will be parameterized(invulnerable to SQL injection) but they will not be prepared(cached in the database session). Not using prepared statements will noticeably decrease performance, so it's recommended to always have this setting enabled.
```